packages/opds-web-client/src/components/DownloadButton.tsx

```diff
@@ -75,7 +76,7 @@ export default class DownloadButton extends React.Component<
       return this.props.fulfill(this.props.url).then(blob => {
         download(
           blob,
-          this.generateFilename(this.props.title),
+          generateFilename(this.props.title, this.fileExtension()),
```

Comment:
This is fine but the reference to the original `generateFilename` function is still in this file and can be removed.